packages/std-client/src/hooks.ts

```diff
@@ -3,6 +3,8 @@ import { ObservableSet } from "mobx";
 import { useEffect, useMemo, useState } from "react";
 import { filter, Observable } from "rxjs";
 
+// Deprecated! Use hooks exported from @latticexyz/react package instead.
+
```

Comment:
```suggestion
// @deprecated Use hooks exported from @latticexyz/react package instead.
```

Using the official tag to integrate with vscode and others (vscode highlights usages of deprecated functions by striking them through)